build-and-run-automaton-core.sh

```diff
@@ -1,5 +1,5 @@
 #/bin/bash
 cd src
 mkdir -p logs
-bazel build --incompatible_package_name_is_a_function=false -c opt //automaton/core && bazel-bin/automaton/core/core # 2>/dev/null
```

Comment:
This will break Mac OS I believe. It likely depends on the bazel version, but this may be necessary.
Did it cause issues for you with it being there?

Reply:
It depends on the bazel version, it was needed with a previous one, but this option does not exist in the newest version

Reply:
Ok then we will just require the newer version of bazel for this.